fabric-connector/src/test/java/com/luxoft/fabric/integration/FabricConnectorIntegrationTest.java

```diff
@@ -24,40 +22,38 @@
  *
  * Sets up environment using default configuration from "files/fabric.yaml"
  */
-public class FabricConnectorIntegrationTest {
+public class FabricConnectorIntegrationTest extends BaseIntegrationTest {
 
-    private static FabricConfig fabricConfig;
     private static final Logger LOG = LoggerFactory.getLogger(FabricConnectorIntegrationTest.class);
```

Comment:
I thank that was yours. `LOG` of `LOGGER`?

Reply:
unified